src/frontend/komponenter/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -1,99 +1,79 @@
 import type { IMinimalFagsak } from '../../../typer/fagsak';
 import type { ITilbakekrevingsbehandling } from '../../../typer/tilbakekrevingsbehandling';
-import {
-    Tilbakekrevingsbehandlingstype,
-    tilbakekrevingstyper,
-} from '../../../typer/tilbakekrevingsbehandling';
-import { datoformat, formaterIsoDato } from '../../../utils/formatter';
+import { Behandlingsresultatstype } from '../../../typer/tilbakekrevingsbehandling';
 import { kalenderDiff } from '../../../utils/kalender';
+import { Behandling } from './Behandling';
+import { BehandlingEllerTilbakebetaling } from './BehandlingEllerTilbakebetaling';
 import type { VisningBehandling } from './visningBehandling';
 
 interface IBehandlingshistorikkProps {
     minimalFagsak: IMinimalFagsak;
 }
 
-const lagLenkePåType = (
-    fagsakId: number,
-    behandling: VisningBehandling | ITilbakekrevingsbehandling
-): ReactNode =>
-    behandling.status === BehandlingStatus.AVSLUTTET ? (
-        behandlingstyper[behandling.type].navn
-    ) : tilbakekrevingstyper.some(type => type === behandling.type) ? (
-        <Lenke
-            href={`/redirect/familie-tilbake/fagsystem/BA/fagsak/${fagsakId}/behandling/${behandling.behandlingId}`}
-            onMouseDown={e => e.preventDefault()}
-            target="_blank"
-        >
-            <span>{behandlingstyper[behandling.type].navn}</span>
-            <ExternalLink />
-        </Lenke>
-    ) : (
-        <Lenke href={`/fagsak/${fagsakId}/${behandling.behandlingId}`}>
-            {behandlingstyper[behandling.type].navn}
-        </Lenke>
-    );
+const konverterBehandling = (
+    faktiskObjekt: VisningBehandling | ITilbakekrevingsbehandling,
+    behandlingEllerTilbakemelding: BehandlingEllerTilbakebetaling
+): BehandlingTabellobjekt => {
+    return {
+        behandlingEllerTilbakemelding: behandlingEllerTilbakemelding,
+        faktiskObjekt: faktiskObjekt,
+    };
+};
 
-const lagLenkePåResultat = (
-    minimalFagsak: IMinimalFagsak,
-    behandling: VisningBehandling | ITilbakekrevingsbehandling
-): ReactNode =>
-    !behandling.resultat ? (
-        '-'
-    ) : tilbakekrevingstyper.some(type => type === behandling.type) ? (
-        <Lenke
-            href={`/redirect/familie-tilbake/fagsystem/BA/fagsak/${minimalFagsak.id}/behandling/${behandling.behandlingId}`}
-            onMouseDown={e => e.preventDefault()}
-            target="_blank"
-        >
-            <span>{behandlingsresultater[behandling.resultat]}</span>
-            <ExternalLink />
-        </Lenke>
-    ) : behandling.status === BehandlingStatus.AVSLUTTET ? (
-        <Lenke href={`/fagsak/${minimalFagsak.id}/${behandling.behandlingId}`}>
-            {behandling ? behandlingsresultater[behandling.resultat] : '-'}
-        </Lenke>
-    ) : (
-        behandlingsresultater[behandling.resultat]
-    );
+interface BehandlingTabellobjekt {
+    behandlingEllerTilbakemelding: BehandlingEllerTilbakebetaling;
```

Comment:
Tilbakekreving er det tekniske begrepet fag bruker, så tenker vi burde holde oss til det
```suggestion
    behandlingEllerTilbakekreving: BehandlingEllerTilbakekreving;
```

Reply:
Eventuelt behandlingEllerTilbakekrevingType, så kan interfacet hete BehandlingEllerTilbakekreving